lib/sntools.js

```diff
@@ -365,76 +365,72 @@ class SNTools {
     readNext();
   }
 
-  convertSimplenoteFiles(files, completion) {
-    let index = 0;
-    let processedData = [];
-
-    let tags = new Map();
+  convertSimplenoteFiles(rawNotes) {
+    // Final notes array
+    let notes = [];
```

Comment:
Could be const? Also we can rename `notes` to `finalNotes` and remove the above comment.